maintenance/wikia/setWikiFactoryVariable.php

```diff
@@ -104,8 +115,16 @@ public function execute() {
 
 			$status = true;
 			if ( $set ) {
-				echo "Set {$this->varName} to ".var_export( $varValue, true );
+				echo "Set {$this->varName} to " . var_export( $varValue, true );
 				$status = $this->setVariable( $id, $varValue );
+			} else if ( $append ) {
+				echo "Appending " . $varValue . " to {$this->varName}:" . PHP_EOL;
+				$varData = (array) WikiFactory::getVarByName( $this->varName, $id, true );
+				$prevValue = unserialize($varData['cv_value']);
+				$newValue = $prevValue . $varValue;
```

Comment:
If we're going to do this a lot, it would be safer to convert to an array, and merge new values (so it will overwrite/update values) instead of just appending possible duplicates to the string.  But I guess this is okay for now.